### DISCUSSION TITLE
Bro, could you change the recipe for "Steel Wall"? please

### DISCUSSION THREAD

**Discussion by Geogit12** [HbmMods/Hbm-s-Nuclear-Tech-GIT#476] [General]
plissss

**Comment by SoB1nary** [comment]
what? why?

**Comment by 9Valjew** [comment]
1. Use nomorerecipeconflict or whatever its called
2. Don't use vmw
3. Skill issue

**Reply by Geogit12** [reply]
xd I do use that mod bro

**Reply by ncr-t** [reply]
that looks like a problem on your end mate

**Reply by HbmMods** [reply]
Then what's the issue, literally just use NMRC as intended

**Reply by Geogit12** [reply]
ah, ok I'll use that mod, thanks for letting me know <3

**Comment by MientusOfficial** [comment]
why
